scripts/installer/install.sh

```diff
@@ -315,7 +315,7 @@ onl_baseplatform="${onl_platform%-r*}"
 echo "BISDN Linux Installer"
 
 # part_size: BISDN Linux partition in MB
-part_size=6144
+part_size=${BISDN_LINUX_VOLUME_SIZE_MB:-6144}
```

Comment:
Maybe move this to top where we also set `BISDN_LINUX_VOLUME_LABEL`, then it will be just overridden when including the platform.conf, and you don't need the if-not-set thingy here.